app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -159,7 +164,15 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                         .edit()
                         .putInt(SHOW_UNPIN_TOAST_COUNTER_PREF, prefInt + 1)
                         .apply()
-                Toast.makeText(context, R.string.homescreen_unpin_tutorial_toast, Toast.LENGTH_LONG).show()
+
+                val contextReference = WeakReference(context)
+                val showToast = postDelayed@{
```

Comment:
Nit: We don't always delay this one, only when VV is enabled.  Might be better to name it `showToast`, same as the val?  Or something similar.